android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsViewModel.kt

```diff
@@ -54,6 +64,57 @@ class SurveyQuestionsViewModel(
     }
 
     fun answerQuestion(questionAnswer: Pair<Int, List<AnswerInput>>) {
-        Timber.d("question ${questionAnswer.first} - answers ${questionAnswer.second}")
+        val uiModels = _questionContentUiModels.value.toMutableList()
+        val changedUiModel = uiModels[questionAnswer.first]
+        changedUiModel.displayType.update(questionAnswer.second)
+        uiModels[questionAnswer.first] = changedUiModel
```

Comment:
Just a suggestion:

```suggestion
        val (index, content) = questionAnswer
        val changedUiModel = uiModels[index]
        changedUiModel.displayType.update(content)
        uiModels[index] = changedUiModel
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/dfd30f14a98fe541a6cf2e25328309b3a9cacbc9